### PR TITLE
[AdminBundle] Fix js error in media chooser

### PR DESCRIPTION
**Pull request by jockri**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets |

### CODE REVIEW

**Comment by krispypen** [comment]
@diskwriter can you have a look? @jockri wrote some javascript ;)

File: src/Kunstmaan/AdminBundle/Resources/ui/js/_auto-collapse-tabs.js [hunk 1]
```diff
@@ -37,7 +37,8 @@ kunstmaanbundles.autoCollapseTabs = (function($, window, undefined) {
                 window.history.pushState({}, null, replaceUrlParam(window.location.href, 'currenttab', activeTab));
             }
 
-            if ($('#currenttab').length > 0) {
+            var currentTabElement = $('#currenttab');
```
Comment (diskwriter) [inline src/Kunstmaan/AdminBundle/Resources/ui/js/_auto-collapse-tabs.js]: currentTabElement can be reused here. No need to reselect.

**Comment by diskwriter** [comment]
Nitpicking. Sorry @jockri  ;)

**Comment by jockri** [comment]
fixed

**Comment by krispypen** [comment]
@roderik :+1:

**Pull request merged**